### DISCUSSION TITLE
How to re-obtain/retreive the Nextcloud AIO admin password

### DISCUSSION THREAD

**Discussion by renni** [nextcloud/all-in-one#4663] [Questions] [answered]
I know the AIO interface on port 8080 or 8443 should show it. But it does not. So how can I retrieve it or reset it?

**Comment by Zoey2936** [comment]
https://github.com/nextcloud/all-in-one/discussions/1786

**Comment by renni** [comment, marked as answer]
I had trioed that. It did not help. But looking at that filwe I found out this is the correct one:

sudo grep NEXTCLOUD_PASSWORD /var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data/data/configuration.json

**Reply by quochung2310** [reply]
Thank you a million! This is the only command worked for AIO docker.
You saved my day!
Best!
